src/Symfony/Bundle/FrameworkBundle/Tests/Routing/RedirectableUrlMatcherTest.php

```diff
@@ -53,7 +53,7 @@ public function testSchemeRedirect()
                 'scheme' => 'https',
                 'httpPort' => $context->getHttpPort(),
                 'httpsPort' => $context->getHttpsPort(),
-                '_route' => 'foo',
+                '_route' => null,
```

Comment:
this change breaks the 3.4 test suite, can't we don without?

Reply:
We don't know the route that would match,  unless we do a new match like for trailing slash. Also the way it worked was not reliable anyway. When you have a condition that didn't match because of the scheme but now matches with a different scheme, this route can be before the route that caused the redirect. So the actual route that will match might not be the one that was passed here at all.

Reply:
we still need to make the currently filing test green before merging...

Reply:
fixed :)